gnovm/stdlibs/io/io_test.gno

```diff
@@ -1,9 +1,9 @@
+package io
```

Comment:
supposed to be io_test

Reply:
Adding the `_test` suffix to packages causes a `... already exists in cache` panic, so I temporarily removed it, but forgot to change it.